sdk/communication/azure-communication-administration/src/test/java/com/azure/communication/administration/PhoneNumberIntegrationTestBase.java

```diff
@@ -10,7 +10,6 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-import com.azure.communication.common.ConnectionString;
```

Comment:
Why not 
`import com.azure.communication.common.implementation.ConnectionString;` ?